### PR TITLE
Sqlhelper and InstanceSettings Helper integration test fixes

### PR DESCRIPTION
**Pull request by jake-martin**

### CODE REVIEW

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/ProcessingHelperTests.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 namespace Helpers.Tests.Integration.Tests
 {
+	//These tests must be run in a specific order
```
Comment (aarongilbert123) [inline CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/ProcessingHelperTests.cs]: Is this still true?

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/SqlHelperTests.cs [hunk 2]
```diff
@@ -52,12 +57,26 @@ public void GetFileShareResourceServerArtifactIdTest()
...
+				Console.WriteLine(ex.Message);
+			}
+			finally
+			{
+				WorkspaceHelper.DeleteSingleWorkspaceAsync(workspaceId);
+			}
 
-			// Act
-			// Assert
-			Assert.DoesNotThrow(() => Sut.EnableDataGridOnExtractedText(Constants.Connection.Sql.EDDS_DATABASE, workspaceName));
```
Comment (calimetikcura) [inline CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/SqlHelperTests.cs]: Why did we delete this code if the test was to test it specifically.

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/SqlHelperTests.cs [hunk 3]
```diff
@@ -52,12 +57,26 @@ public void GetFileShareResourceServerArtifactIdTest()
...
+			try
+			{
+				// Arrange
+				workspaceId = WorkspaceHelper.CreateSingleWorkspaceAsync(Constants.Workspace.DEFAULT_WORKSPACE_TEMPLATE_NAME, workspaceName, true).Result;
+
+				// Act
+				// Assert
+				Assert.DoesNotThrow(() => Sut.EnableDataGridOnExtractedText(Constants.Connection.Sql.EDDS_DATABASE, workspaceName));
+			}
+			catch (Exception ex)
```
Comment (calimetikcura) [inline CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/SqlHelperTests.cs]: no catch blocks similar to previous comment.

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/InstanceSettingsHelperTests.cs [hunk 4]
```diff
@@ -31,18 +33,29 @@ public void TearDown()
...
+				// Assert
+				Assert.True(_createdInstanceSettingId != 0);
+			}
+			catch (Exception ex)
+			{
+				System.Console.WriteLine(ex.Message);
+			}
+			finally
+			{
+				Sut.DeleteInstanceSetting(_createdInstanceSettingId);
```
[the remark this answers is not in the document]
Comment (jake-martin) [reply CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/InstanceSettingsHelperTests.cs]: Checking if the Id is not 0 before deleting. This should be the only necessary check

File: CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/InstanceSettingsHelperTests.cs [hunk 5]
```diff
@@ -51,14 +64,26 @@ public void UpdateInstanceSettingTest()
...
+				// Assert
+				Assert.AreEqual(value, instanceSettingValue);
+			}
+			catch (Exception ex)
+			{
+				System.Console.WriteLine(ex.Message);
+			}
+			finally
+			{
+				Sut.UpdateInstanceSettingValue(name, section, "");
```
[the remark this answers is not in the document]
Comment (jake-martin) [reply CSharp/DevVmPowershell/Helpers.Tests.Integration/Tests/InstanceSettingsHelperTests.cs]: done

**Pull request merged**